docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs

```diff
@@ -2,14 +2,14 @@
 using System.Linq;
 using Microsoft.ML.Data;
 
-namespace Microsoft.ML.Samples.Dynamic
+namespace Microsoft.ML.Samples.Dynamic.TensorFlow
 {
-    class TensorFlowTransformExample
+    class ImageClassification
     {
         /// <summary>
-        /// Example use of the TensorFlowEstimator in a ML.NET pipeline.
+        /// Example use of the TensorFlow image model in a ML.NET pipeline.
         /// </summary>
-        public static void TensorFlowScoringSample()
+        public static void ScoringWithImageClassificationModelSample()
```

Comment:
>ScoringWithImageClassificationModelSample [](start = 27, length = 41)

I would assume we have references on this file in documentation for Catalog extension methods. like
```
        /// <example>
        /// <format type="text/markdown">
        /// <![CDATA[
        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
        /// ]]>
        /// </format>
        /// </example>
```
So you need to update that reference as well

Reply:
Thanks! Did a quick search and found this one the only reference in the solution. Fixed it.

---
In reply to: [254125126](https://github.com/dotnet/machinelearning/pull/2429#discussion_r254125126) [](ancestors = 254125126)